scripts/uninstall

```diff
@@ -29,6 +29,8 @@ source "$BASE_DIR/mesh-point/uninstall"
 source "$BASE_DIR/hostapd/uninstall"
 source "$BASE_DIR/ipfs/uninstall"
 source "$BASE_DIR/extra-tools/uninstall"
+source "$BASE_DIR/node_exporter/uninstall"
+source "$BASE_DIR/prometheus/uninstall"
```

Comment:
Ha! you know i remember wondering why grafana was installed on time after i uninstalled it.... that would explain it.. 

Added uninstall script